docs/quick-visual-tutorials/visual-k8s-cluster-mapping.mdx

```diff
@@ -143,17 +143,15 @@ We also (as a default) told Otterize Cloud that there is a global default-deny n
 
 #### Blocking status
 
-Note that the locks themselves are green, indicating that you could now turn on enforcement without blocking any clients.
+Note that the locks themselves are yellow, indicating that you could now turn on enforcement and blocking not intented clients.
```

Comment:
```suggestion
Note that the services themselves are yellow, indicating that access would be blocked if protection were enabled.
```
@sapirwo Yellow actually means would be blocked, so it's not safe to turn on enforcement at this point!